### PR TITLE
Move tracers first order flux to new tendency specification

### PR DESCRIPTION
**Pull request by charleskawczynski**
### Description

This PR moves tracers first order flux to the new tendency specification.

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: src/Atmos/Model/declare_prognostic_vars.jl [hunk 1]
```diff
@@ -18,4 +18,4 @@ abstract type Precipitation <: PrognosticVariable end
 struct Rain <: Precipitation end
 struct Snow <: Precipitation end
 
-struct Tracers <: PrognosticVariable end
+struct Tracers{N} <: PrognosticVariable end
```
Comment (kpamnany) [inline src/Atmos/Model/declare_prognostic_vars.jl:21]: Presumably `N` is the number of tracers? I'm not seeing it used though?

Comment (charleskawczynski) [reply src/Atmos/Model/declare_prognostic_vars.jl:21]: Yes, should we add a doc string? It's not really being used, but it shows up in the tendency table.

Comment (kpamnany) [reply src/Atmos/Model/declare_prognostic_vars.jl:21]: It can be added when it's used.

**Review by kpamnany — APPROVED** [review]

**Pull request merged**